src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Listing/Listing.ts

```diff
@@ -134,6 +137,8 @@ export class Listing<Container extends ResourcesBase.IResource> {
 
                 $scope.createPath = adhPreliminaryNames.nextPreliminary();
 
+                $scope.noCounterValue = typeof ($scope.counterValue) === "undefined";
```

Comment:
This is not updated when `$scope.conterValue` changes. What about a function like this:

```js
$scope.getCounterValue = () => {
    return typeof $scope.counterValue !== "undefined" ? $scope.counterValue : $scope.totalCount;
};
```

This would also allow to reduce duplication in the template.